ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4409,7 +4417,130 @@ HWDDC:
   implements:
   - CONTROL
 
+# New abstracts for MFA BV100 HVAC
+CFDPM:
+  description: "Carbon filter pressure monitoring."
```

Comment:
Please add to description, something like: "Carbon filter pressure monitoring, where specific filter type is required."